client/state/selectors/is-nav-unification-enabled.js

```diff
@@ -12,10 +12,37 @@ import { getSelectedSiteId } from 'calypso/state/ui/selectors';
 import { isJetpackSite } from 'calypso/state/sites/selectors';
 import isAtomicSite from 'calypso/state/selectors/is-site-automated-transfer';
 import { getCurrentUserId } from 'calypso/state/current-user/selectors';
+import isNavUnificationNewUser from 'calypso/state/selectors/is-nav-unification-new-user';
 
 // Gradual rollout (segment of existing users + all new users registered after March 5, 2021).
-const NEW_USER_ID_THRESHOLD = 202731080; // ID of user who first registered on March 5, 2021.
 const CURRENT_ROLLOUT_SEGMENT_PERCENTAGE = 5;
+// Test users found in test/e2e/config/local-decrypted.json.
+const TEST_USER_IDS = [
+	114387625,
+	196072882,
+	147903583,
+	176665678,
+	191685058,
+	99044787,
+	115575915,
+	125513142,
+	140566456,
+	139474794,
+	139474794,
+	102365391,
+	106276422,
+	106276977,
+	107524077,
+	107524337,
+	107524381,
+	107524493,
+	107524451,
+	107523085,
+	127372741,
+	123892165,
+	126716843,
+	132724203,
+];
```

Comment:
Can we replace this with the `isE2ETest` util? 

https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/e2e/index.ts

Reply:
I now feel so dumb about my static IDs 😄 ... f94b968